Userland/Libraries/LibCrypto/Hash/SHA1.cpp

```diff
@@ -67,6 +83,188 @@ inline void SHA1::transform(u8 const* data)
     secure_zero(blocks, 16 * sizeof(u32));
 }
 
+#if ARCH(I386) || ARCH(X86_64)
+sha1_attribute_target_x86 static void transform_impl(u32* state, u8 const* data)
+{
+    __m128i reverse_8;
+    __m128i abcd_1;
+    __m128i e;
+    __m128i old_abcd;
+    __m128i old_e;
+    __m128i msg_0;
+    __m128i abcd_2;
+    __m128i msg_1;
+    __m128i msg_2;
+    __m128i msg_3;
```

Comment:
This is still very C.

1) Initialize all primitive type variables. 
2) Only define variables at first use, not at the top of the function. We aren't in C89 anymore

Reply:
OK, will create variables at theirs first use.

Reply:
Fixed.